src/views/bouquets/BouquetFormView.vue

```diff
@@ -43,12 +42,14 @@ const {
 
 const topic: Ref<Partial<TopicPostData> & Pick<TopicPostData, 'extras'>> = ref({
```

Comment:
Pour corriger  le [fixme](https://github.com/opendatateam/udata-front-kit/pull/661/files#diff-2c444ced84d1ea003ef370c47ba2fef24deab5d8e4f7f431ce7cfb56f2be37b6R248)
```suggestion
const topic: Ref<Partial<TopicPostData> & Pick<TopicPostData, 'extras' | 'tags'>> = ref({
```